service/worker/migration/activities.go

```diff
@@ -372,9 +373,20 @@ func (a *activities) generateWorkflowReplicationTask(
 
 	var stateTransitionCount, historyLength int64
 	if generateViaFrontend {
+		archetype, ok := a.chasmRegistry.ComponentFqnByID(execution.ArchetypeId)
+		if !ok {
+			err := fmt.Errorf("unknown archetypeID: %v", execution.ArchetypeId)
+			a.logger.Error("force-replication failed to translate archetypeID to name", tag.Error(err), tag.ArchetypeID(execution.ArchetypeId))
```

Comment:
Put the execution key in the log message to help debug this?

Reply:
Done.